mlrun/db/httpdb.py

```diff
@@ -4209,6 +4209,11 @@ def store_alert_config(
         if not alert_data:
             raise mlrun.errors.MLRunInvalidArgumentError("Alert data must be provided")
 
+        if mlrun.mlconf.alerts.mode == mlrun.common.schemas.alert.AlertsModes.disabled:
+            logger.warning(
+                "Alerts are disabled, alert will still be stored but will not be triggered"
+            )
+
```

Comment:
Should add a similar check & warning to the `generate_event` method - since it will be disabled, but will not fail, on the client-side there will be no indication to the situation, and the debug log in the server side will not be visible.